src/App.js

```diff
@@ -0,0 +1,71 @@
+import { useState } from "react";
+import { Container } from "./Components/Container.styled";
+
+import { Body } from "./Components/Body.styled";
+
+import WeatherLoader from "./Components/WeatherLoader";
+
+const api = {
+  key: "4ff50176f17a1a9eec162b0203a0abad",
+  base: "https://api.openweathermap.org/data/2.5/",
+};
+
+function App() {
+  const [search, setSearch] = useState("");
+  const [weather, setWeather] = useState({});
+  const [background, setBackground] = useState("");
+  const [isLoading, setIsLoading] = useState(false);
+
+  const searchPressed = () => {
```

Comment:
rename this to searchHandler

Reply:
also consider using try catch and async await like we did in valina js example of this app